cpp/src/arrow/util/bit-util.h

```diff
@@ -430,12 +345,7 @@ class BitmapReader {
     }
   }
 
-#if defined(_MSC_VER)
-  // MSVC is finicky about this cast
   bool IsSet() const { return (current_byte_ & (1 << bit_offset_)) != 0; }
```

Comment:
Any idea if this will be slower on gcc/clang?

Reply:
That sounds unlikely to me.